Sources/Sentry/SentryProfiler.mm

```diff
@@ -415,26 +462,85 @@ + (void)useFramesTracker:(SentryFramesTracker *)framesTracker
 
 #    pragma mark - Private
 
-+ (void)captureEnvelopeIfFinished:(SentryProfiler *)profiler spanID:(SentrySpanId *)spanID
++ (NSArray *)slicedArray:(NSArray *)array transaction:(SentryTransaction *)transaction
```

Comment:
`l`: We could only pass in the `startSystemTime` and `endSystemTime` of the `transaction`.

Reply:
Let me skip this one for now, and circle back to it for a later refactor of SentryProfiler I have planned.